shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/NonTraceProperties/Validity.hs

```diff
@@ -56,7 +56,9 @@ data ValidationError
   | -- | The transaction fee is too small
     FeeTooSmall Coin Coin
   | -- | Value is not conserved
-    ValueNotConserved Coin Coin
+    ValueNotConserved
+  | -- | ada being forged
+    UserForgingAda
```

Comment:
Did we specify that this is an error case? I thought it sufficed that the Ada policy ID isn't a hash.

Reply:
Yeah. there is a special check. but i added the error anyways.